### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by bored2020** [abess-team/abess#259]
When I import abess in each python version I try,such as ( 3.5  3.6  3.7  ) all report the error as follow'
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\__init__.py", line 9, in <module>
    from abess.linear import abessLogistic, abessLm, abessCox, abessPoisson, abessMultigaussian, abessMultinomial
  File "C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\linear.py", line 3, in <module>
    from .bess_base import bess_base
  File "C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\bess_base.py", line 7, in <module>
    from abess.cabess import pywrap_abess
  File "C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\cabess.py", line 13, in <module>
    from . import _cabess
ImportError: DLL load failed: 找不到指定的模块。
```

I try all I can find in the web,but not works, cloud anyone can help me .

**Comment by bored2020** [comment]
In the jupyter the more detail as follows:

---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
c:\Users\Administrator\Documents\WeChat Files\g853346463\FileStorage\File\2021-12\luwenwenjian\datasets\huizong\huizong\Breast\abessfenxi.py in <module>
----> 12 import abess

C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\__init__.py in <module>
      7 
      8 # from abess.linear import PdasLm, PdasLogistic, PdasPoisson, PdasCox, L0L2Lm, L0L2Logistic, L0L2Poisson, L0L2Cox, GroupPdasLm, GroupPdasLogistic, GroupPdasPoisson, GroupPdasCox, abessLogistic
----> 9 from abess.linear import abessLogistic, abessLm, abessCox, abessPoisson, abessMultigaussian, abessMultinomial
     10 from abess.pca import abessPCA
     11 from abess.datasets import make_glm_data, make_multivariate_glm_data

C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\linear.py in <module>
      1 
      2 from abess.metrics import concordance_index_censored
----> 3 from .bess_base import bess_base
      4 
      5 import numpy as np

C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\bess_base.py in <module>
      5 from sklearn.utils.validation import check_X_y, check_array, check_is_fitted
      6 from sklearn.base import BaseEstimator
----> 7 from abess.cabess import pywrap_abess
      8 
      9 

C:\ProgramData\Anaconda3\envs\cp3710\lib\site-packages\abess\cabess.py in <module>
     11 # Import the low-level C/C++ module
     12 if __package__ or "." in __name__:
---> 13     from . import _cabess
     14 else:
     15     import _cabess

ImportError: DLL load failed: 找不到指定的模块。

**Comment by bored2020** [comment]
Yes, I tried for pyhton3.9 it still report this error,but in python3.8 it's ok.But I have to use some other packages below the python3.8, so I have to ask for help.Thanks a lot.

**Comment by Jiang-Kangkang** [comment]
Could you show me the install information when you install abess?

**Comment by bored2020** [comment]
I have this platform  information 
```
uname_result(system='Windows', node='PC-20211105RHLF', release='10', version='10.0.19041', machine='AMD64', processor='Intel64 Family 6 Model 165 Stepping 3, GenuineIntel')
python version :'3.7.10'
```

and I `pip uninstall abess` and install it again,has the following information
```
(cp3710) C:\Users\Administrator>pip install abess
Collecting abess
  Using cached abess-0.3.0-cp37-cp37m-win_amd64.whl (426 kB)
Requirement already satisfied: sklearn in c:\programdata\anaconda3\envs\cp3710\lib\site-packages (from abess) (0.0)
Requirement already satisfied: scipy in c:\programdata\anaconda3\envs\cp3710\lib\site-packages (from abess) (1.7.3)
Requirement already satisfied: numpy in c:\programdata\anaconda3\envs\cp3710\lib\site-packages (from abess) (1.21.4)
Requirement already satisfied: scikit-learn in c:\programdata\anaconda3\envs\cp3710\lib\site-packages (from sklearn->abess) (1.0.1)
Requirement already satisfied: threadpoolctl>=2.0.0 in c:\programdata\anaconda3\envs\cp3710\lib\site-packages (from scikit-learn->sklearn->abess) (3.0.0)
Requirement already satisfied: joblib>=0.11 in c:\programdata\anaconda3\envs\cp3710\lib\site-packages (from scikit-learn->sklearn->abess) (1.1.0)
Installing collected packages: abess
Successfully installed abess-0.3.0

```

**Comment by bored2020** [comment]
Finally solve this problem in my windows computer,it may because  I don't have `MinGW-w64` in my path,but in `pip install abess` there is no error report,may be can put the  `MinGW-w64`  in the requirement.

Thanks for both of you ,thanks a lot.

**Comment by Mamba413** [comment]
> Finally solve this problem in my windows computer,it may because I don't have `MinGW-w64` in my path,but in `pip install abess` there is no error report,may be can put the `MinGW-w64` in the requirement.
> 
> Thanks for both of you ,thanks a lot.

Congratulation! You solve this problem by yourself before we exactly reproduce the error appeared in your personal computer.  We are very appreciate for your suggestion, and we may include your suggestion into our documentation. 

BTW, why you install abess from our source code? You are going to develop novel new features?

**Comment by bored2020** [comment]
Thanks,but I want to use the abess method to solve some real problem by now,but because I don't use the `C and  C++`well,I want  to develop some features it's diffcult for me ,but I try to make it.

**Comment by Mamba413** [comment]
> Thanks,but I want to use the abess method to solve some real problem by now,but because I don't use the `C and C++`well,I want to develop some features it's diffcult for me ,but I try to make it.

Thanks for your response. If you still have any question when developing code, please feel free to contact us.

**Comment by Mamba413** [comment]
Hi, we have fixed this issue in version 0.4.0. We expect this package can be installed via `pip install abess` in Windows platform. @bored2020

**Comment by bored2020** [comment]
Thanks a lot.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "abess-team/abess"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年1月22日(星期六) 晚上11:42
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [abess-team/abess] Import error (Issue #259)





 
Hi, we have fixed this issue in version 0.4.0. We expect this package can be installed via pip install abess in Windows platform. @bored2020
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
